src/features/pastelSentTxStore/sentTxStore.ts

```diff
@@ -1,23 +1,19 @@
-import { remote } from 'electron'
+import { app, remote } from 'electron'
 import fs from 'fs'
 import os from 'os'
 import path from 'path'
-
-import {
-  TDetailedTxns,
-  TListTransactions,
-  TSentTxStore,
-  TVjoinsplit,
-} from '../pastelDB/type'
+import { TSentTxStore, TTransaction } from 'types/rpc'
 
 const locateSentTxStore = (): string => {
+  const electronApp = app || remote.app
```

Comment:
I don't think this is the right way. Please take a look here: 

https://github.com/pastelnetwork/pastel-electron-wallet/blob/master/src/main.ts#L250

https://github.com/pastelnetwork/pastel-electron-wallet/blob/master/src/main.ts#L108

https://github.com/pastelnetwork/pastel-electron-wallet/blob/master/src/renderer.tsx#L69